extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -15,6 +15,8 @@ import { TelemetryActions, TelemetryReporter, TelemetryViews } from '../common/t
 export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined, packageHelper: PackageHelper): Promise<void> {
 	TelemetryReporter.sendActionEvent(TelemetryViews.SqlBindingsQuickPick, TelemetryActions.startAddSqlBinding);
 
+	const vscodeMssqlApi = await utils.getVscodeMssqlApi();
```

Comment:
the sql bindings unit tests are failing with these changes, so make sure to update those, too.

Reply:
Unit tests should be fixed now. See a PR validation issue but maybe transient.